Makefile

```diff
@@ -15,13 +15,24 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
+OPA_IMAGE := opa
```

Comment:
Should we use the image we maintain? https://github.com/forseti-security/policy-library/blob/master/cloudbuild.yaml

Reply:
Let me try using that one. With the official OPA image, `fmt -d` does not work because `diff` cannot be found. Maybe it will work with this image.

Reply:
I'm not running it right.
```
docker run -it --rm \
                -v $(CURDIR):/workspace \
                gcr.io/config-validator/rego-v0.16.0:latest \
                test
```
produces error:
```
make: *** No rule to make target 'test'.  Stop.
make: *** [docker_test] Error 2
```

Reply:
This command worked for me, you need to set the working directory (or we could update the Dockerfile to include WORKDIR):
```
docker run -it --rm \
  -v "$(pwd):/workspace" \
  -w=/workspace \
  gcr.io/config-validator/rego-v0.16.0:latest \
  test
```

Reply:
Nice, thanks! This works for me now.